tests/features/S2N_CLOEXEC_XOPEN_SUPPORTED.flags

```diff
@@ -0,0 +1 @@
+-Werror
```

Comment:
remind me why we need `-Werror` here. I see some `.flags` files have this and other dont

Reply:
some probes might want to narrow the scope of the errors to particular warnings, rather than getting "unused variables" or "unused functions", etc.

The thinking here is the probe is small enough that you wouldn't get any other warnings from anything but the include.

Reply:
Looking at [code](https://github.com/aws/s2n-tls/blob/1640db5d07b465fe6790e8829909b939dd8c1b33/CMakeLists.txt#L334), it seems like GLOBAL_FLAGS and PROBE_FLAGS are both used in the try_compile.

To me it seems like the flags are additive rather than narrowing. Am I miss-reading cmake here?

Reply:
That's correct; they are narrowing. But you may not want `-Werror` included globally, since it would be more difficult to change on an individual probe.

Reply:
It is additive.

You also still have to be careful of "unused variables" or you'll break make ;) https://github.com/aws/s2n-tls/issues/4187